### PR TITLE
Finished ✨  local storage to mark favorite coins ✨ 

### PR DESCRIPTION
**Pull request by bejanaf**
- changed the kind of fetch
- added a new useEffect
- added a new useState
- added a new toggle function
- added a new ternary to style favorite icon
- mapped favorite coins
-> to test local storage

- fixed position of navigation and and made the content visible until the end on news.js 
- removed unused properties
- removed unused styling in NewsCard.js

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -54,6 +69,20 @@ function App() {
...
+        topCoin.isFavorite = !topCoin.isFavorite;
+      }
+      return topCoin;
+    });
+    setTopCoins(likedCoins);
+    const favoriteCoins = topCoins.filter((topCoin) => topCoin.isFavorite);
+    setFavoriteCoins(favoriteCoins);
+  }
+
+  console.log('Zeig mir die Favoriten:', favoriteCoins);
```
Comment (SabMueller) [inline client/src/App.js]: you know what to do here ... :)

File: client/src/components/CoinsCard.js [hunk 2]
```diff
@@ -72,6 +85,14 @@ const CoinImage = styled.img`
   width: 1.1rem;
 `;
 
+const CoinName = styled.span``;
+
+const FavoriteIcon = styled.img`
+  width: 1.1rem;
```
Comment (SabMueller) [inline client/src/components/CoinsCard.js]: consider sorting alphabetically :)

File: client/src/components/CoinsCard.js [hunk 3]
```diff
@@ -72,6 +85,14 @@ const CoinImage = styled.img`
   width: 1.1rem;
 `;
 
+const CoinName = styled.span``;
```
Comment (SabMueller) [inline client/src/components/CoinsCard.js]: you shouldn't use empty styles ... :) if you dont need it, remove it :D

File: client/src/pages/Wallet.js [hunk 4]
```diff
@@ -1,10 +1,18 @@
...
 
-export default function Wallet() {
+export default function Wallet({ favoriteCoins }) {
   return (
     <>
       <Headliner>Wallet</Headliner>
+      {favoriteCoins.map((favoriteCoin) => (
+        <WalletWrapper>
+          <span>
+            LocalStorage test, disyplay name of favorite coin:{' '}
```
Comment (SabMueller) [inline client/src/pages/Wallet.js]: please remove the test text :)

**Review by SabMueller — APPROVED** [review]
looks good to me ❤️  ✨  love your handling of the favorites!

File: client/src/App.js (hunk 1 above)
Comment (bejanaf) [reply client/src/App.js]: yessss, I removed them! :D

File: client/src/components/CoinsCard.js (hunk 3 above)
Comment (bejanaf) [reply client/src/components/CoinsCard.js]: removed :)

File: client/src/components/CoinsCard.js (hunk 2 above)
Comment (bejanaf) [reply client/src/components/CoinsCard.js]: fixed :)

File: client/src/pages/Wallet.js (hunk 4 above)
Comment (bejanaf) [reply client/src/pages/Wallet.js]: removed :)

**Comment by bejanaf** [comment]
Thank you very much for your code-review and handsome comments ❤️ 😄

**Pull request merged**